src/it/default-incremental-disable/pom.xml

```diff
@@ -30,14 +30,6 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
-  <dependencies>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>3.8.2</version>
-    </dependency>
-  </dependencies>
-
```

Comment:
Could we do that same here? I.e., replace JUnit 3 by JUnit 5, then add a dummy method annotated with `@Test` in the `MyTest` class below.